query-engine/core/src/telemetry/capturing/mod.rs

```diff
@@ -161,10 +164,7 @@ pub fn capturer(trace_id: trace::TraceId, settings: Settings) -> Capturer {
 #[cfg(feature = "metrics")]
 #[allow(clippy::type_complexity)]
 pub fn install_capturing_layer(
-    subscriber: Layered<
-        Option<query_engine_metrics::MetricRegistry>,
-        Layered<Box<dyn Layer<Registry> + Send + Sync>, Registry>,
-    >,
+    subscriber: impl SubscriberExt + for<'a> LookupSpan<'a> + Send + Sync + 'static,
```

Comment:
Made it generic instead of hardcoding the huge nested specific type, which now changed because the metrics layer doesn't exist anymore.